docs/developer-guide/ksqldb-reference/scalar-functions.md

```diff
@@ -413,6 +414,20 @@ the entire substring is returned by default.
 For example, `REGEXP_EXTRACT("(.*) (.*)", 'hello there', 2)`
 returns "there".
 
+REGEXP_EXTRACT_ALL
```

Comment:
When you resolve the merge conflicts, this will become ### REGEXP_EXTRACT_ALL.

Reply:
Done.